src/Native/libunwind/src/DwarfInstructions.hpp

```diff
@@ -186,10 +187,12 @@ int DwarfInstructions<A, R>::stepWithDwarf(A &addressSpace, pint_t pc,
                 i, getSavedVectorRegister(addressSpace, registers, cfa,
                                           prolog.savedRegisters[i]));
           else if (i == (int)cieInfo.returnAddressRegister) {
-            pint_t dummyLocation;
             returnAddress = getSavedRegister(addressSpace, registers, cfa,
                                              prolog.savedRegisters[i],
-                                             dummyLocation);
+                                             returnAddressLocation);
+            if (registers.validRegister(i)) {
```

Comment:
What is the situation where this returns false here?

Reply:
On CPUs that don't have a physical register to store the return address. 